ui/src/wide_series_stat.tsx

```diff
@@ -28,12 +28,12 @@ const
       padding: grid.gap,
     },
     plot: {
-      display: 'flex',
-      width: pc(50),
+      position: 'relative',
+      width: '100%',
     },
     text: {
-      width: pc(50),
       paddingLeft: grid.gap,
+      alignSelf: 'center',
```

Comment:
I am thinking that centering may not be the best idea here. Wdyt?

<img width="295" alt="image" src="https://user-images.githubusercontent.com/64769322/191028195-a2f9f2e9-d2f6-446d-aba4-e8ab53a36527.png">

vs 

<img width="305" alt="image" src="https://user-images.githubusercontent.com/64769322/191028246-7f89d36f-34ba-48a9-880c-9f776c7542f4.png">

Reply:
Why not? It fills the available space and also it is consistent with a gauge stat. Do you find the first one more elegant or is there any issue with that?

Reply:
No issue, just cosmetics. Let's leave it as is. We can revisit in future if needed.